### DISCUSSION TITLE
Running html file on Github

### DISCUSSION THREAD

**Discussion by keijiota** [jspsych/jsPsych#3176] [Q&A]
Hello JsPsych community, 

Thanks as always for your support. I wonder if there is a way to run html file on Github rather than just viewing the source of the file. The html file contains JsPsych materials. So, essentially I would like to run JsPsych code on my Github repository. Would that be possible? 

Many thanks,
Keiji

**Comment by jodeleeuw** [comment]
Hi @keijiota,

Yes, check out [GitHub Pages](https://pages.github.com/). This lets you run directly on GitHub and is easy to configure.
